pkg/operator/operator.go

```diff
@@ -452,6 +464,75 @@ func (c *csiDriverOperator) isCSIDriverInUse() (bool, error) {
 	return false, nil
 }
 
+func (c *csiDriverOperator) checkPrereq(instance *v1alpha1.AWSEBSDriver) error {
+	return c.checkPreviousInstall(instance)
+}
+
+func (c *csiDriverOperator) checkPreviousInstall(instance *v1alpha1.AWSEBSDriver) error {
+	// Check that there is no other CSI driver installation.
+	// Get the driver name
+	expectedDriver := resourceread.ReadCSIDriverV1Beta1OrDie(generated.MustAsset(csiDriver))
+	driverName := expectedDriver.Name
+	if err := c.checkPreviousCSIDriver(driverName); err != nil {
+		return err
+	}
+
+	// CSIDriver does not exist. Check if the operand namespace exists.
+	_, err := c.namespaceInformer.Lister().Get(operandNamespace)
```

Comment:
Do we need new rbac permissions?

Reply:
not for namespaces, they're already included in RBAC